app/views/newflow/student_signup/student_email_verification_form.html.erb

```diff
@@ -0,0 +1,40 @@
+<%
+    page_header = I18n.t(
```

Comment:
consider:  building up a data structure in the controller that has all this stuff and exposing a single PORO to this form for all its data needs (instead of building up this giant set of data inside the view.   it's usually best to keep views thin (any rails-y layer)

Reply:
I love that idea! That would be a great next step for the newflow views 👍 . I do personally adopt the rule "make it work, then make it better" though. This makes it work _and_ it's already a huge improvement over how it was in the old flow, IMO. So, my hope is that already improved code like this will reveal patterns that can **later** then be implemented even more uniformly at a higher level.